TLM/TLM/UI/MainMenu/MainMenuWindow.cs

```diff
@@ -537,13 +409,13 @@ private struct AddButtonsResult {
                 // might not exist
                 StatsLabel.enabled = Options.showPathFindStats && value;
             }
+
             UResizer.UpdateControl(this);
         }
 
         public override void OnDestroy() {
             eventVisibilityChanged -= OnVisibilityChanged;
             confDisposable?.Dispose();
-            base.OnDestroy();
         }
```

Comment:
Why did you remove that method call? Without it `BaseUWindowPanel` will be leaking memory from not disposed observables.

Reply:
Reverted this line